tests/unit_tests/shared/test_port_handler.py

```diff
@@ -335,7 +335,7 @@ def test_reuse_active_live_nok_nok(unused_tcp_port):
         )
 
     with pytest.raises(port_handler.NoPortsInRangeException):
-        _, port, sock = port_handler.find_available_port(
+        _, port, _ = port_handler.find_available_port(
```

Comment:
I think we should rewrite this port_handler method. Shouldn't `find_available_port` only return the available port? I will create an issue for this.

Reply:
https://github.com/equinor/ert/issues/8407